code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -770,19 +788,634 @@ nobiliumsuppression = INFINITY
 		location = get_turf(pick(pipenet.members))
 	else
 		location = get_turf(holder)
-	air.assert_gases(/datum/gas/water_vapor,/datum/gas/nitryl,/datum/gas/carbon_dioxide,/datum/gas/nitrogen)
-	var/ball_shot_angle = 180*cos(cached_gases[/datum/gas/water_vapor][MOLES]/cached_gases[/datum/gas/nitryl][MOLES])+180
-	var/stim_used = min(STIM_BALL_GAS_AMOUNT/cached_gases[/datum/gas/plasma][MOLES],cached_gases[/datum/gas/stimulum][MOLES])
-	var/pluox_used = min(STIM_BALL_GAS_AMOUNT/cached_gases[/datum/gas/plasma][MOLES],cached_gases[/datum/gas/pluoxium][MOLES])
-	var/energy_released = stim_used*STIMULUM_HEAT_SCALE//Stimulum has a lot of stored energy, and breaking it up releases some of it
+	air.assert_gases(/datum/gas/water_vapor, /datum/gas/nitryl, /datum/gas/carbon_dioxide, /datum/gas/nitrogen)
+	var/ball_shot_angle = 180 * cos(cached_gases[/datum/gas/water_vapor][MOLES] / cached_gases[/datum/gas/nitryl][MOLES]) + 180
+	var/stim_used = min(STIM_BALL_GAS_AMOUNT / cached_gases[/datum/gas/plasma][MOLES], cached_gases[/datum/gas/stimulum][MOLES])
+	var/pluox_used = min(STIM_BALL_GAS_AMOUNT / cached_gases[/datum/gas/plasma][MOLES], cached_gases[/datum/gas/pluoxium][MOLES])
+	var/energy_released = stim_used * STIMULUM_HEAT_SCALE//Stimulum has a lot of stored energy, and breaking it up releases some of it
 	location.fire_nuclear_particle(ball_shot_angle)
-	cached_gases[/datum/gas/carbon_dioxide][MOLES] += 4*pluox_used
-	cached_gases[/datum/gas/nitrogen][MOLES] += 8*stim_used
+	cached_gases[/datum/gas/carbon_dioxide][MOLES] += 0.5 * pluox_used
+	cached_gases[/datum/gas/nitrogen][MOLES] += 2 * stim_used
 	cached_gases[/datum/gas/pluoxium][MOLES] -= pluox_used
 	cached_gases[/datum/gas/stimulum][MOLES] -= stim_used
-	cached_gases[/datum/gas/plasma][MOLES] *= 0.5 //Consumes half the plasma each time.
+	cached_gases[/datum/gas/plasma][MOLES] -= min(stim_used * pluox_used, 10)
```

Comment:
won't this cause issues?

Reply:
not many people know this exists, even less know how to pull this off and less know how to make it shoot where you want (math and whatnot) halving plasma heat tick causes the reaction to be over before it was even put into uses. I could make the number in the min higher (25 or 40 even) but i doubt there will be much problems (and if there will be i'll just revert this one)

Reply:
aight sounds good to me, if you're gonna worry about negative numbers how does this behave?

Reply:
Right I'll have to add a chexk